scheds/rust/scx_layered/src/main.rs

```diff
@@ -2282,6 +2292,36 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn refresh_idle_qos(&mut self) -> Result<()> {
+        if !self.idle_qos_enabled {
+            return Ok(());
+        }
+
+        for layer in self.layers.iter() {
+            let idle_resume_us = layer.kind.common().idle_resume_us as i32;
+            for cpu in 0..layer.cpus.len() {
+                if layer.cpus.test_cpu(cpu) {
+                    update_cpu_idle_resume_latency(cpu, 0)?;
+                    if idle_resume_us > 0 {
+                        update_cpu_idle_resume_latency(cpu, idle_resume_us)?;
+                    }
```

Comment:
Why the double updates?

Reply:
Was being silly, forgot there can only be one request at a time from the userspace API.